wnprc_billing/resources/views/updateProgramIncomeAccount.html

```diff
@@ -0,0 +1,130 @@
+        <tr>
+            <td style="padding:5px">JETCSVSetting</td>
+            <td style="padding:5px" id="jetcsvCell1"></td>
+            <td style="padding:5px" id="jetcsvCell2">
+                <form>
+                    <input type="text" id="field2" name="field2">
+                </form>
+            </td>
+            <td style="padding:5px">
+                <button id="field2Button" onclick="updateField2()">Update JETCSVSetting</button>
+            </td>
+        </tr>
+    </table>
+
+    <script>
+        function updateField1() {
+            let field1NewValue = document.getElementById("field1").value;
+            setValues("CreditToAccount", "" + field1NewValue);
+        }
+
+        function updateField2() {
+            let field2NewValue = document.getElementById("field2").value;
+            setValues("JETCSVSetting", "" + field2NewValue);
+        }
+
+        //This function updates the view with the current values for 'CreditToAccount' and 'JetCsvSetting'.
+        function getValues() {
+            LABKEY.Ajax.request({
+                url: LABKEY.ActionURL.buildURL('wnprc_billing', 'getProgramIncomeAccount', null, {}),
+                callback: function (config, success, xhr) {
+                    if (success) {
+                        var jsonResponse = JSON.parse(xhr.responseText);
+                        var currentCreditToAccount = jsonResponse.creditToAccount;
+                        var currentJETCSVSetting = jsonResponse.jetCSVSetting;
+                        document.getElementById("ctaCell1").innerHTML = currentCreditToAccount;
+                        document.getElementById("jetcsvCell1").innerHTML = currentJETCSVSetting;
+                    }
+                    else {
+                        alert('Couldn\'t get program income account.');
+                    }
+                }
+            });
+        }
+
+        //This function verifies the user has the EHRFinanceAdminPermission, then sets the new
+        // values for 'CreditToAccount' and 'JetCsvSetting'.
+        function setValues(propertyName, newValue) {
+            if ((propertyName !== "") && (newValue !== "")) {
+                //Verifies the current user has EHRFinanceAdminPermission.
+                LABKEY.Ajax.request({
+                    url: LABKEY.ActionURL.buildURL('wnprc_billing', 'checkFinanceAdminStatus', null, {}),
+                    callback: function (config, success, xhr) {
+                        if (success) {
+                            //Sets the new module property values.
+                            let parentContainer = LABKEY.Security.currentContainer.parentId;
+                            let newPropertyValue = newValue;
+                            let propertyToUpdate = propertyName;
+                            let moduleToUpdate = "wnprc_billing"
+
+                            //Creates an object with the new module property values.
+                            var newProperties = [];
```

Comment:
any reason for mixing "var" and "let"? in general, it's good to start using "let" and "const" afaik, I would suggest trying to use let since var can introduce weird bugs with scoping. I don't think it causes issues here, just good practice.

Reply:
I'll keep an eye on this; I always forget that you can still append an array declared with 'let' or 'const'.